src/jit/compiler.hpp

```diff
@@ -3592,7 +3592,7 @@ inline CORINFO_METHOD_HANDLE Compiler::eeFindHelper(unsigned helper)
     /* Helpers are marked by the fact that they are odd numbers
      * force this to be an odd number (will shift it back to extract) */
 
-    return ((CORINFO_METHOD_HANDLE)(size_t)((helper << 2) + 1));
+    return ((CORINFO_METHOD_HANDLE)((((size_t)helper) << 2) + 1));
```

Comment:
It would be preferable to use C++ style casts rather than C style casts if you're doing this kind of thing.

Reply:
I agree with that, but I think it should be part of this particular PR.